includes/functions/functions_general.php

```diff
@@ -1616,6 +1616,10 @@ function zen_update_record_company_clicked($recordCompanyId, $languageId)
     $sql = $db->bindVars($sql, ':languageId:', $languageId, 'integer');
     $db->execute($sql);
   }
+
+  function issetorArray(array $array, $key, $default) {
```

Comment:
$default needs to be optional:
```php
function safe_array_value(array $anArray, $aKey, $aDefault = null) {
```
Also needs a docblock.
Note: php7 will have a short hand that will make this obsolete:  `var_dump($array[$key] ?? $default);`